lib/rules/no-empty-first-line/__tests__/index.js

```diff
@@ -0,0 +1,87 @@
+"use strict";
+
+const rule = require("..");
+const { messages, ruleName } = rule;
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+  skipBasicChecks: true,
+  fix: true,
+
+  accept: [
+    {
```

Comment:
Can we get tests for single-line and multiline whitespace, please? e.g.

```js
  {
    code: "  "
  },
  {
    code: "\n"
  },
  {
    code: "\n\n"
  },
```

To test [this line](https://github.com/stylelint/stylelint/pull/3650/files#diff-0465d1dd14c45b6f34815530242119e8R26).